packages/pattern-commands/README.md

```diff
@@ -0,0 +1,122 @@
+<div align="center">
+
+![Sapphire Logo](https://cdn.skyra.pw/gh-assets/sapphire-banner.png)
+
+# @sapphire/plugin-pattern-commands
+
+**Plugin for <a href="https://github.com/sapphiredev/framework">@sapphire/framework</a> so you can have pattern commands.**
+
+[![GitHub](https://img.shields.io/github/license/sapphiredev/plugins)](https://github.com/sapphiredev/plugins/blob/main/LICENSE.md)
+[![codecov](https://codecov.io/gh/sapphiredev/plugins/branch/main/graph/badge.svg?token=QWL8FB16BR)](https://codecov.io/gh/sapphiredev/plugins)
+[![npm bundle size](https://img.shields.io/bundlephobia/min/@sapphire/plugin-pattern-commands?logo=webpack&style=flat-square)](https://bundlephobia.com/result?p=@sapphire/plugin-pattern-commands)
+[![npm](https://img.shields.io/npm/v/@sapphire/plugin-pattern-commands?color=crimson&logo=npm&style=flat-square)](https://www.npmjs.com/package/@sapphire/plugin-pattern-commands)
+
+</div>
+
+## Description
+
+With pattern commands you can define rulesets to which the bot may respond.
+
+## Important notes
+
+-   From April 2022 this plugin will only work with Message Intent
```

Comment:
```suggestion
-   Starting April 2022 this plugin will only work with Message Intent. This is because this plugin relies on scanning messages, which is impossible without the intent.
```